addons/hr_holidays/tests/test_accrual_allocations.py

```diff
@@ -0,0 +1,237 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import datetime, time
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields
+
+from odoo.tools import mute_logger
+
+from odoo.addons.hr_holidays.tests.common import TestHrHolidaysBase
+
+
+class TestAccrualAllocations(TestHrHolidaysBase):
+    def setUp(self):
+        super(TestAccrualAllocations, self).setUp()
+
+        # Make sure we have the rights to create, validate and delete the leaves, leave types and allocations
+        self.LeaveType = self.env['hr.leave.type'].sudo(self.user_hrmanager_id).with_context(tracking_disable=True)
+        self.Allocation = self.env['hr.leave.allocation'].sudo(self.user_hrmanager_id).with_context(tracking_disable=True)
```

Comment:
I know that @tde-banana-odoo does like setting configured model on `self` in the setup. This is more explicit to do a variable in each test.